src/frontend/planner_test/tests/testdata/nexmark.yaml

```diff
@@ -1153,38 +1153,36 @@
-    └─StreamProject { exprs: [auction.id, auction.item_name, count(bid.auction)] }
-      └─StreamDynamicFilter { predicate: (count(bid.auction) >= $expr1), output: [auction.id, auction.item_name, count(bid.auction), count(bid.auction)] }
-        ├─StreamProject { exprs: [auction.id, auction.item_name, count(bid.auction), count(bid.auction)] }
-        | └─StreamHashAgg { group_key: [auction.id, auction.item_name], aggs: [count, count(bid.auction), count(bid.auction)] }
-        |   └─StreamHashJoin { type: Inner, predicate: auction.id = bid.auction, output: all }
-        |     ├─StreamExchange { dist: HashShard(auction.id) }
-        |     | └─StreamTableScan { table: auction, columns: [auction.id, auction.item_name], pk: [auction.id], dist: UpstreamHashShard(auction.id) }
-        |     └─StreamExchange { dist: HashShard(bid.auction) }
-        |       └─StreamTableScan { table: bid, columns: [bid.auction, bid._row_id], pk: [bid._row_id], dist: UpstreamHashShard(bid._row_id) }
-        └─StreamExchange { dist: Broadcast }
-          └─StreamProject { exprs: [(sum0(count) / count(bid.auction)) as $expr1] }
-            └─StreamGlobalSimpleAgg { aggs: [count, sum0(count), count(bid.auction)] }
-              └─StreamExchange { dist: Single }
-                └─StreamProject { exprs: [bid.auction, count] }
-                  └─StreamAppendOnlyHashAgg { group_key: [bid.auction], aggs: [count, count] }
-                    └─StreamExchange { dist: HashShard(bid.auction) }
-                      └─StreamTableScan { table: bid, columns: [bid.auction, bid._row_id], pk: [bid._row_id], dist: UpstreamHashShard(bid._row_id) }
+    └─StreamDynamicFilter { predicate: (count(bid.auction) >= $expr1), output: [auction.id, auction.item_name, count(bid.auction)] }
+      ├─StreamProject { exprs: [auction.id, auction.item_name, count(bid.auction)] }
+      | └─StreamHashAgg { group_key: [auction.id, auction.item_name], aggs: [count, count(bid.auction)] }
+      |   └─StreamHashJoin { type: Inner, predicate: auction.id = bid.auction, output: all }
+      |     ├─StreamExchange { dist: HashShard(auction.id) }
+      |     | └─StreamTableScan { table: auction, columns: [auction.id, auction.item_name], pk: [auction.id], dist: UpstreamHashShard(auction.id) }
+      |     └─StreamExchange { dist: HashShard(bid.auction) }
+      |       └─StreamTableScan { table: bid, columns: [bid.auction, bid._row_id], pk: [bid._row_id], dist: UpstreamHashShard(bid._row_id) }
+      └─StreamExchange { dist: Broadcast }
+        └─StreamProject { exprs: [(sum0(count) / count(bid.auction)) as $expr1] }
+          └─StreamGlobalSimpleAgg { aggs: [count, sum0(count), count(bid.auction)] }
+            └─StreamExchange { dist: Single }
+              └─StreamProject { exprs: [bid.auction, count] }
+                └─StreamAppendOnlyHashAgg { group_key: [bid.auction], aggs: [count, count] }
```

Comment:
Seems we still have 2 `count`s here? 👀

Reply:
That's because the internal `count` inserted when building streaming plan. Will get rid of it later (#8197).